shared/admin-tools/elasticsearch/find-cases-missing-practitioners.js

```diff
@@ -0,0 +1,228 @@
+// how to run
```

Comment:
Can we add this to a readme?

Reply:
We absolutely can!

However, I have a suspicion that, given this script took nearly 5 hours to execute, it is probably not worth keeping around once [#9323](https://github.com/flexion/ef-cms/issues/9323) is addressed.

Reply:
I don't want to "solution" that bug, but I'd imagine the bones of this script could potentially be used as the basis for a migration that generates the missing entities and then this script could be deleted.

Reply:
Oh, that makes sense.